xx.md

```diff
@@ -0,0 +1,46 @@
+NIP-XX
+======
+
+Audio Events
+---------------
+
+`draft` `optional`
+
+This NIP defines a new event kind `31337` representing an audio track. These events are _parameterized replaceable_
+and deletable per [NIP-09](09.md).
+
+Unlike a `kind 1` event with audio attached, audio events are intended to be surfaced in an audio-specific context
+rather than in a general micro-blogging context.
+
+A kind `31337` event's `content` should be summary or description of the audio content.
+
+The following tags are required:
+
+- `d` is a unique identifier. This MAY be generated by the client creating the event, or refer to an external GUID.
+- `media` is the url of the media. This SHOULD NOT be read, but is required for backwards compatibility reasons.
+- `imeta` is as described in NIP 92. Multiple `imeta` tags MAY be included to provide for different use cases
+  (e.g. streaming vs download).
+- `subject` is the title of the audio track
+
+The following tags are optional:
+
+- `website` is an external url to a website related to the track.
+- `creator` is the pubkey (or plain text name) of the track author(s).
+- `duration` is the duration of the audio track, in seconds.
+- `published_at` is a timestamp representing the track's original publish date.
+
+```json
+{
+  "id": <event_id>,
+  "pubkey": <author_pubkey>,
+  "created_at": <created_at>,
+  "kind": 31337,
+  "content": "Chill beats",
+  "tags": [
+    ["d", "<UUID>"],
+    ["subject", "Platinum Robots on the Moon"],
```

Comment:
Why not `title` to keep it consistent with NIP-23 and others?

Reply:
I'm trying to keep this compatible with zapstr.live which uses subject. Also, subject is defined in nip 14 so it's not a terrible choice.

Reply:
IMO `title` and `subject` are different things. Title is generally a name. Subject is a very short "what's this about".

Songs have title, not subject.

Reply:
Yes, I agree, but there are lots of events in the wild already that use subject.

Reply:
I've updated this to use `title` but require `subject` for backwards compatibility with zapstr.

Reply:
> I'm trying to keep this compatible with zapstr.live which uses subject. Also, subject is defined in nip 14 so it's not a terrible choice.

NIP-14 is why I used `subject` at the time 😅